tasks/get_record.json

```diff
@@ -1,20 +1,16 @@
 {
     "puppet_task_version": 1,
     "supports_noop": false,
-    "description": "Read a ServiceNow object",
+    "description": "Get a ServiceNow record\nbolt task run --nodes dev84270 servicenow_tasks::get_record table=incident fields='{\"sysid\"=\"3Db9b91557db00101096dde37a48961976\"}'",
     "remote": true,
     "files": ["ruby_task_helper/files/task_helper.rb"],
     "parameters": {
         "table": {
             "description": "ServiceNow table",
             "type": "Optional[String[1]]"
         },
-        "number": {
-            "description": "ServiceNow number",
-            "type": "Optional[String[1]]"
-        },
-        "lookup_field": {
-            "description": "ServiceNow lookup_field",
+        "fields": {
+            "description": "ServiceNow fields to query by",
             "type": "Optional[String[1]]"
```

Comment:
This also shouldn't be `Optional`. Also, this should probably be a hash.

Reply:
made it mandatory hash.